components/core/src/clp/streaming_archive/ArchiveMetadata.hpp

```diff
@@ -102,6 +145,9 @@ class ArchiveMetadata {
     // The size of the archive
     uint64_t m_compressed_size{0};
     uint64_t m_dynamic_compressed_size{0};
+    std::string m_variable_encoding_methods_version{ffi::cVariableEncodingMethodsVersion};
```

Comment:
The new coding standard encourages using std::string_view where possible. is it possible to use string_view instead?